.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs: {}
+  push:
+    tags:
+      - 'v*'
+jobs:
+  publish-chart:
+    name: Release new Kaptain-Catalog-Applications version
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: self-hosted
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Create tarball
+        run: |
+          tar -zcf kaptain-catalog-applications.tar.gz --owner root --group root .
```

Comment:
subject to review

Reply:
I would say it's up to you, the main goal is to have a link that we can include to Kaptain release notes so `kaptain-catalog-applications` tarball can be downloaded from there for the air-gapped installation.

Reply:
yeah. let's go with the manual generation for now.
I've tested that if we use this command to create the tarball and then use step 5 on [this page](https://docs.d2iq.com/dkap/2.1/add-kaptain-to-dkp-catalog-applications-on-air-g-1#id-(2.1)AddKaptaintoDKPCatalogApplicationsonAir-gappedEnvironmentsforDKP2.2andlater-LoadtheDockerimagesintoyourDockerregistry) the contents get unzipped with no issues